index.html

```diff
@@ -3,6 +3,7 @@
 <head>
   <title>Fingerprintjs2 test</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!--
```

Comment:
undo this.

Reply:
in China. Not Open Google

Reply:
I can undo it before push action every time.

Reply:
-------------------------------
How does the Silverlight and Flash detection relate to the existing plugin detection?

What are your thoughts on cookies?
-------------------------------
The First Q&A: Maybe plugin detection contains Silverlight and Flash, but donot have version.
The Second Q&A: I think browser cookie function whether to enable,Can be use user environmental risk judgment